src/types/fetchers.ts

```diff
@@ -57,14 +57,17 @@ export enum FetcherName {
   TWAPGasPrice = 'TWAPGasPrice',
   OnChainData = 'OnChainData',
   UniswapV3 = 'UniswapV3',
+  UniswapV3OLD = 'UniswapV3Fetcher',
   UniswapPrice = 'UniswapPrice',
+  UniswapPriceOLD = 'UniswapPriceFetcher',
   GoldApiPrice = 'GoldApiPrice',
   MetalPriceApi = 'MetalPriceApi',
   MetalsDevApi = 'MetalsDevApi',
   OptionsPrice = 'OptionsPrice',
   YearnVaultTokenPrice = 'YearnVaultTokenPrice',
   RandomNumber = 'RandomNumber',
   SovrynPrice = 'SovrynPrice',
+  SovrynPriceOLD = 'SovrynPriceFetcher',
   ByBitPrice = 'ByBitPrice',
   BinancePrice = 'BinancePrice',
```

Comment:
ByBit and Binance even they changed, they don't require changes since they were not enabled.